CODE/Sources/PennyBOT/Constants.swift

```diff
@@ -3,8 +3,10 @@ import DiscordBM
 
 enum Constants {
     static let internalChannelId = "441327731486097429"
+    static let botDevUserId = "290483761559240704"
```

Comment:
My own user id, which the logs will mention on `error`/`warning` or when the bot restarts.
We can change it to a role too, if you want some other people to be notified as well.